workflows/mcmicro.nf

```diff
@@ -88,6 +89,17 @@ workflow MCMICRO {
     }
     */
 
+    // Run Coreograph
+    if (params.tma_dearray) {
+        COREOGRAPH(ASHLAR.out.tif)
+        COREOGRAPH.out.cores
+            .transpose()
+            .map { meta, img -> [[id: meta.id + '_' + img.toString().split('/')[-1].tokenize(".")[0]], img]}
```

Comment:
I believe `img` is a `Path` object here, so using the `Path` API to obtain the filename would read more cleanly:
```groovy
img.fileName.toString().tokenize('.')[0]
```

Reply:
Ah, of course, that's much cleaner.  Thanks!!